dockerfiles/Dockerfile.jvm

```diff
@@ -4,40 +4,44 @@ LABEL maintainer="Gary Tierney <gary.tierney@digirati.com>"
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 ARG OPENJDK_11_VERSION=11.0.3.7-2.el8_0
+ARG OPEN_SSL_VERSION=1.1.1-8.el8
 
 RUN microdnf install \
-       java-11-openjdk-devel-$OPENJDK_11_VERSION \
-    && microdnf clean all
+       java-11-openjdk-devel-$OPENJDK_11_VERSION openssl-$OPEN_SSL_VERSION \
+    && microdnf clean all \
+    && mkdir -p /build
+
+WORKDIR /build
 
 ENV JAVA_HOME="/usr/lib/jvm/java"
 ENV GRADLE_OPTS="-Dorg.gradle.caching=false -Dorg.gradle.daemon=false -Dorg.gradle.parallel=true"
 
-RUN mkdir -p /build/
-WORKDIR /build/
-
 COPY build.gradle.kts settings.gradle.kts gradlew gradle.properties ./
 COPY gradle ./gradle/
 COPY taxonomy-manager-rest ./taxonomy-manager-rest/
 COPY taxonomy-manager-rest-server ./taxonomy-manager-rest-server/
 COPY taxonomy-manager-common ./taxonomy-manager-common/
 COPY taxonomy-manager-engine ./taxonomy-manager-engine/
+
 RUN ./gradlew assemble && rm -Rf "$HOME/.gradle"
 
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.0 AS runtime
 
 ARG OPENJDK_11_VERSION=11.0.3.7-2.el8_0
 
-RUN mkdir -p /srv/taxman/
-WORKDIR /srv/taxman/
+RUN microdnf install \
+       java-11-openjdk-headless-$OPENJDK_11_VERSION \
```

Comment:
There are two ways to approach this, either:

1. The `runtime` stage is based on the `build` base layer (i.e. `FROM build`), in which case we get everything that's in that base layer; or
2. We use `copy` to copy artifacts into the `runtime` stage from the `build `layer.

Neither are going to be any use to us because `1` will include **everything** from the `build` layer, including all the build specific cruft that we don't want in `runtime`, and `2` requires you know the location of every file, symbolic link, modification, etc that `microdnf install java-11-openjdk-headless` carries out so that we can target each and every file and replicate each and every modification. This is obviously non-trivial and a waste of time.

The `copy` command is designed to copy artifacts either from another layer in a multi-stage build, or from the local file system during build, that form part of the "custom" solution that the image is being built for.